storage/paths/db_index.go

```diff
@@ -312,6 +313,8 @@ func (dbi *DBIndex) StorageDetach(ctx context.Context, id storiface.ID, url stri
 }
 
 func (dbi *DBIndex) StorageReportHealth(ctx context.Context, id storiface.ID, report storiface.HealthReport) error {
+	ctx = metrics.AddNetworkTag(ctx)
```

Comment:
I'm still concerned about these miscellaneous calls that receive a context - I'd rather find the source of the context if we can and deal with it there. I think that this one is for `api.StorageMiner` which is used by `node.MinerHandler` in the RPC initialising code. And you're already taking care of it there. I think you might be able to start up a `lotus-miner` and run the same `curl` to see what metrics it has and see whether the tag is necessary here or not.

Are we dealing with metrics in here that are showing up without a tag in your existing `curl`? Because as far as I know these are only called by a `lotus-miner` server process.

Reply:
@virajbhartiya remove all the changes in the storage/ packages and let's ship this without them, I think it's good to go otherwise (although I haven't tested it locally yet but I will do once these are removed)